### PR TITLE
Update misc.py

### PR DESCRIPTION
**Pull request by tenuuni**
added /epochalypse and /host_time

### CODE REVIEW

File: cogs/cmds/misc.py [hunk 1]
```diff
@@ -23,6 +23,8 @@
 import datetime
 import humanfriendly
 import importlib
+from datetime import date
```
Comment (0vf) [inline cogs/cmds/misc.py:26]: Duplicate import (we already import datetime)

**Review by 0vf — CHANGES_REQUESTED** [review]
LGTM, all that's needed is to remove the duplicate import and it should be ready to merge

File: cogs/cmds/misc.py [hunk 2]
```diff
@@ -69,7 +71,39 @@ async def uptime(self,interaction: discord.Interaction):
 		timedelta = datetime.datetime.now(datetime.UTC) - self.bot.start_time
 		await interaction.response.send_message(content=f"{humanfriendly.format_timespan(timedelta)}")
 
+
+	@app_commands.command(description="Get time from host")
+	async def host_time(self,interaction: discord.Interaction):
+		today = date.today()
```
Comment (0vf) [inline cogs/cmds/misc.py:77]:
```suggestion
		today = datetime.date.today()
```

File: cogs/cmds/misc.py (hunk 1 above)
Comment (0vf) [inline cogs/cmds/misc.py:26]:
```suggestion
```

**Review by tenuuni — COMMENTED** [review]
aright

**Pull request merged**